tomviz/python/AutoTiltAxisRotationAlignment.py

```diff
@@ -87,8 +87,9 @@ def transform_scalars(self, dataset):
         rot_ang = coarseAngles[minIntensityIndex]
 
         self.progress.message = 'Rotating tilt series'
-        result = ndimage.interpolation.rotate(
-            tiltSeries, -rot_ang, axes=((0, 1)))
+        result = np.empty_like(tiltSeries)
```

Comment:
@yijiang1 Looking at the [doc](https://docs.scipy.org/doc/scipy-0.16.1/reference/generated/scipy.ndimage.interpolation.rotate.html) for ndimage.interpolation.rotate, it will reshape the output array if necessary.